mne/externals/FieldTrip.py

```diff
@@ -309,7 +309,7 @@ def getHeader(self):
                 (chunk_type, chunk_len) = struct.unpack(
                     'II', payload[offset:offset + 8])
                 offset += 8
-                if offset + chunk_len < bufsize:
+                if offset + chunk_len > bufsize:

```

Comment:
@jjnurminen can you explain this change? It seems to work fine without this change on my machine. I did:
```sh
$ neuromag2ft --file test_elekta_raw.fif
```
and then ran then ran `ftclient_rt_average.py` till line 57. You can do:

```py
>>> raw_info['ch_names']
```